README.md

```diff
@@ -134,6 +135,11 @@ Want to deploy yourself with Render? Click the button below:
 
 [![Deploy to Render](https://render.com/images/deploy-to-render-button.svg)](https://render.com/deploy?repo=https://github.com/OWASP/wrongsecrets)
 
+### Running these on Railway
+
+If you want to host WrongSecrets on Railway, you can do so by deploying [this one-click template](https://railway.app/new/template/7pnwRj), or by clicking the button below. Railway does not offer an always-free plan anymore, but the free trial is good enough to test-drive this before you decide to upgrade. If you need a step-by-step companion guide, see [this blog post](https://alphasec.io/test-your-secret-management-skills-with-owasp-wrongsecrets/).
+
+[![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/new/template/7pnwRj)
```

Comment:
Can we make this a link instead please? After 5 bucks people need to pay, so I rather advertise it less (okteto and intended are truly free, hence they have buttons ;-))

Reply:
Certainly, we can use your suggestion.